Cargo.toml

```diff
@@ -47,6 +47,7 @@ stm32f3 = "0.13.2"
 bxcan = { version = ">=0.4, <0.6", optional = true }
 stm32-usbd = { version = "0.6.0", optional = true }
 void = { version = "1.0.2", default-features = false }
+enumset = { version = "1.0.6", optional = true}
```

Comment:
Reminder to self: Do not forget to mention this feature in 

https://github.com/stm32-rs/stm32f3xx-hal/blob/598fb342c4d64c1b13d5d1147d9e50a236739bb6/src/lib.rs#L50